packages/@react-spectrum/table/docs/TableView.mdx

```diff
@@ -498,6 +498,9 @@ See the docs for more information on how to perform server side sorting.
 
 ```tsx example
 function AsyncSortTable() {
+
+  let collator = useCollator();
+
```

Comment:
You'll need to import `useCollator` here in the example e.g. `import {useCollator} from '@react-aria/i18n';`

Reply:
Added the missing import.